concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
+
+inputs:
+  - name: cf-manifest
+  - name: bosh-secrets
+run:
+  path: sh
+  args:
+  - -c
+  - -e
+  - |
+    bosh_password=$(awk '/bosh_admin_password/ { print $2 }' bosh-secrets/bosh-secrets.yml)
+    bosh -t 10.0.0.6 login admin "${bosh_password}"
```

Comment:
Alex is going to see what this looks like and decide whether it's worth doing now.